src/skybox_and_splitscreen.c

```diff
@@ -584,22 +568,19 @@ void func_802A4D18(void) {
     }
     switch (gScreenModeSelection) {                 /* switch 1; irregular */
     case SCREEN_MODE_1P:                                         /* switch 1 */
-        D_80150148 = 1.333333f;
+        D_80150148 = 1.33333334f;
```

Comment:
Is there a difference between `1.333333f` and `1.33333334f`?

Reply:
IDO seems to think so. `1.3333f` is turned into a double that looks something like `3fccccc9` but in order to match it needs to be `3fccccca`.
I suspect that if I did something like `4.0f/3.0f` that it wouldn't look quite as strange (I think that matches, didn't test it though)